src/complex-list-field/complex-list-field.component.html

```diff
@@ -1,28 +1,43 @@
 <div [id]="path.join('.')">
-  <div *ngFor="let value of values; let i = index; trackBy:trackByFunction" [id]="path.join('.') + '.' + i">
+  <!-- Navigator -->
+  <table *ngIf="navigator" class="navigator-container">
+    <tr>
+      <td class="form-group navigator-item-left">
+        <div class="input-group input-group-sm">
+          <span class="input-group-btn">
+            <button type="button" class="btn btn-default" (click)="onFindClick()">⚲</button>
+          </span>
+          <input type="search" class="form-control" [(ngModel)]="expression" placeholder="Find" />
+          <span class="input-group-btn">
+            <button type="button" class="btn btn-default" [disabled]="currentFound <= 0" (click)="onFoundNavigate(-1)">❮</button>
+          </span>
+          <span class="input-group-btn">
+            <button type="button" class="btn btn-default" [disabled]="currentFound >= foundIndices.length - 1" (click)="onFoundNavigate(1)">❯</button>
+          </span>
+          <span *ngIf="foundIndices.length > 0" class="input-group-addon">
+            {{foundIndices.length}} out of {{values.size}}
```

Comment:
Yes, so what I was proposing above would be to use `currentFound` instead of `foundIndices.length`